### PR TITLE
version increase to 2.3.1

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- increase the version to 2.3.1
- update the changelog

- [x] merge after #154

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.3.1crp
+--------
+**Date:** 2019-02-14
+
+A critical bug was fixed which can result into an illegal memory access.
```
Comment (ax3l) [inline CHANGELOG.md]: please mention the method that is affected

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+### Changes to mallocMC 2.3.0crp
+
+**Bug fixes**
+ - fix illegal memory access #150
```
Comment (ax3l) [inline CHANGELOG.md]: please mention the method that is affected

**Comment by psychocoderHPC** [comment]
@ax3l it is updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+--------
+**Date:** 2019-02-14
+
+A critical bug was fixed which can result in an illegal memory access.
+
+### Changes to mallocMC 2.3.0crp
+
+**Bug fixes**
+ - fix illegal memory access in `XMallocSIMD` #150
+
```
Comment (ax3l) [inline CHANGELOG.md]: Let us add #154 while we are on it :)

Comment (psychocoderHPC) [reply CHANGELOG.md]: Yes, not good if we work not side by side.^^ I will add it to the changelog and do a break to avoid double work^^

**Comment by psychocoderHPC** [comment]
This can be merged

**Pull request merged**